dev/bots/test.dart

```diff
@@ -965,14 +965,23 @@ Future<void> _runFrameworkTests() async {
     );
   }
 
+  // Tests that take longer than average to run. This is usually because they
+  // need to compile something large or make use of the analyzer for the test.
+  Future<void> runSlow() async {
+    printProgress('${green}Running slow package tests$reset for directories other than packages/flutter');
+    await runExampleTests();
```

Comment:
Right now there are three `Linux`, `Mac`, and `Windows framework_tests_misc` shards.  A better way to shard this out would be to only have tests that need to run on one platform so you only need a `Linux framework_tests_slow` (or another name) shard.  Linux VM bots are the most plentiful and cheapest to run on.

So `runExampleTests` needs to validate the tests pass on all platforms, but I believe `runTracingTests`, `runFixTests`, and `runPrivateTests` are platform-agnostic and only need to run once.

Reply:
All of these tests were being run on all platforms previously, as the `misc` sub-shard contained all of these tests and ran on all of the platforms. I personally don't have the history or expertise to confidently say which of these tests should be run on which platform, and this PR is to address a P1 flake, so I'd prefer that to be a followup issue if you believe it's something we should pursue.

Reply:
I am pretty confident that `runTracingTests`, `runFixTests`, and `runPrivateTests` are platform-agnostic.  If you want to run `runExampleTests` then you also need to create `Mac` and `Windows` versions of your builds in `.ci.yaml`.  So either way you need to upload a new diff and wait for the tests to pass to close the P1 🙂

Reply:
Sure thing. `runExampleTests` is now back in the `runMisc` function, so it should run on all platforms. I had planned on adding mac and windows versions of the `slow` sub-shard later after I figured out the needed dependencies, but I'll skip that now since you're certain the remaining tests in the new sub-shard are platform agnostic.